app/views/cloud_volume/new.html.haml

```diff
@@ -6,10 +6,11 @@
         = _('Storage Manager')
       .col-md-8
         = select_tag("storage_manager_id",
-                     options_for_select([["<#{_('Choose')}>", nil]] + @storage_manager_choices.sort, disabled: ["<#{_('Choose')}>", nil]),
+                     options_for_select((@storage_manager_id.nil? ? [["<#{_('Choose')}>", nil]] : []) + @storage_manager_choices.sort, disabled: ["<#{_('Choose')}>", nil]),
```

Comment:
@gberginc This is something similar to what I suggested to @miha-plesko .
If we could use an API to retrieve the Storage Managers, that would be perfect...
(can be totally done in a follow-up PR)